Code/Framework/AzCore/AzCore/IO/Streamer/ReadSplitter.cpp

```diff
@@ -361,10 +365,26 @@ namespace AZ::IO
 
     void ReadSplitter::CollectStatistics(AZStd::vector<Statistic>& statistics) const
     {
-        statistics.push_back(Statistic::CreateFloat(m_name, AvgNumSubReadsName, m_averageNumSubReadsStat.GetAverage()));
-        statistics.push_back(Statistic::CreatePercentage(m_name, AlignedReadsName, m_alignedReadsStat.GetAverage()));
-        statistics.push_back(Statistic::CreateInteger(m_name, NumAvailableBufferSlotsName, aznumeric_caster(m_availableBufferSlots.size())));
-        statistics.push_back(Statistic::CreateInteger(m_name, NumPendingReadsName, aznumeric_caster(m_pendingReads.size())));
+        statistics.push_back(Statistic::CreateFloatRange(
+            m_name, AvgNumSubReadsName, m_averageNumSubReadsStat.GetAverage(), m_averageNumSubReadsStat.GetMinimum(),
+            m_averageNumSubReadsStat.GetMaximum(),
+            "The average number of splits. A value of 3 would for instance indicate that on average files are split into 3 smaller "
+            "reads."));
+        statistics.push_back(Statistic::CreatePercentageRange(
+            m_name, AlignedReadsName, m_alignedReadsStat.GetAverage(), m_alignedReadsStat.GetMinimum(),
+            m_alignedReadsStat.GetMaximum(),
+            "The percentage or reads that meet the alignment requirements and therefore don't need to use a buffer slot. Higher values are "
+            "better as it means more requests that require fewer overhead."));
+        statistics.push_back(Statistic::CreateInteger(
+            m_name, NumAvailableBufferSlotsName, aznumeric_caster(m_availableBufferSlots.size()),
+            "The total number or available slots to buffer unaligned reads to. If this is frequently low, more requests that meet "
+            "alignment requirements help lower it. Alternatively the allocated memory can be increased as that will provide additional "
```

Comment:
This description is confusing.
I don't know if this value should be low or should be high?

Reply:
Reworded the comment in the next commit.